src/modm/driver/can/mcp2515_impl.hpp

```diff
@@ -134,11 +111,12 @@ modm::Mcp2515<SPI, CS, INT>::setFilter(accessor::Flash<uint8_t> filter)
 {
 	using namespace mcp2515;
 
+	while(!this->acquireMaster()){};
```

Comment:
it works, but there is no way for these functions to be resumable ... but the acquireMaster has to be called to configure the spi bus before doing this. You have  not proposed a solution for this problem

Reply:
> it works

As I heard from you and other people this driver actually does not work for "unknown reason"...

You may suggest a solution here just as I can, without a proper solution this won't get merged otherwise.
Furthermore I have proposed a solution in the initial comment to this review (same problem/solution as with `initializeWithPrescaler()`...).